ts/components/PinSelectionComponent.tsx

```diff
@@ -83,16 +83,23 @@ const PinSelectionComponent = ({ navigation, isOnboarding }: Props) => {
   );
 
   return (
-    <BaseScreenComponent
-      goBack={handleGoBack}
-      contextualHelpMarkdown={contextualHelpMarkdown}
-      faqCategories={["onboarding_pin", "unlock"]}
-      headerTitle={I18n.t("onboarding.pin.headerTitle")}
+    <KeyboardAvoidingView
+      keyboardVerticalOffset={0}
+      behavior="padding"
```

Comment:
Do we need this component to avoid hiding the "submit" button? If so I looked to the [official example](https://reactnative.dev/docs/keyboardavoidingview) and I see that set this property in a platform specific way: 

```ts
 behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
```

Do you know why?